### DISCUSSION TITLE
About ASK (Amplitude Shift Keying)

### DISCUSSION THREAD

**Discussion by designe** [euphony-io/euphony#49] [Knowledge Share]
Honestly,
Few software developers actually know ASK. We're busy studying programming, but What's the ASK?!

 ASK is the most basic modulation method in communication. 
It can be seen as a basic method of converting data from the analog world to digital. 

 In order for wireless or wired communication to work, in the end, electrical or radio signals are moved to the outside world through air or wires, and the computer uses a modem to represent them as data.
 
 Radio uses radio waves, AM and FM. This is an abbreviation for Amplitude Modulation and Frequency Modulation. 
As everyone knows about radio quality, you already know from your own experience that FM is good?

![Picture1](https://user-images.githubusercontent.com/1748714/133625997-e0aad9c6-3495-4d44-ab4b-a8084cf140ac.gif)

 ASK is a method of recognizing data through Amplitude. Amplitude in a sound wave corresponds to the intensity volume of the sound wave. If you look above image, you can see that the amplitude periodically rises and falls at 18000hz, right? 
This is the data that is modulated with 0 and 1 :)

**Comment by psh2849** [comment]
Thanks! It was very helpful :+1:

**Comment by seohy02** [comment]
Interesting !!! 😀😀😀
